package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/to-color",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

```suggestion
  "version": "2.1.1",
```

I think this can just be a patched version as no functionality has changed in the build.

---

Changed